dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -736,6 +736,8 @@ public void sortTopLevelCategories()  throws DotDataException {
 	}
 
 	public void sortChildren(String inode)  throws DotDataException {
+        inode = SQLUtil.sanitizeParameter( inode );

```

Comment:
That was my first thought, the thing is that we would need a mejor refactor on that logic, first we need to change [this method](https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/MySQLCategorySQL.java#L22-L28), the signature for every other DBs, in order to return a PreparedStatement not a String, then we would need to change the other methods as well to avoid having 2 patterns in the same functionality. I couldn't find any integration test around that logic and at that point I changed my mind and did the implemented the safer option: SQL Sanitize Parameter. 

@dsilvam and @wezell let me know if you want to use the PreparedStatement anyway and I will develop it and create the tests around it.